org.tango.pogo.pogo_gui/src/org/tango/pogo/pogo_gui/DeviceClass.java

```diff
@@ -429,7 +429,7 @@ public String getSourceFile(String lang) {
             case "cpp":
                 return path + pogoClass.getName() + ".cpp";
             case "python":
-                return path + pogoClass.getName() + ".py";
+            	return path + pogoClass.getName() + ".py";
```

Comment:
If this indentation is changing, then may as well make the indentation in all the `case` clauses the same in this switch statement.

Reply:
Agree. In last pull request some conflict was there in this change. While resolving the conflict, I might put the incorrect indentation. I have updated the code now. Also, incorporated the suggestion in this [merge request ](https://github.com/tango-controls/pogo/commit/ab288f2b5e4e4a2cac37bd7bd49e010420403c83).